packages/editor/src/client/atoms/descriptionListwithMessage.tsx

```diff
@@ -0,0 +1,42 @@
+import React, {ReactNode} from 'react'
+import {InfoMessage, InfoColor} from './infoMessage'
+
+export interface DescriptionListWithMessageItemProps {
+  label?: ReactNode
+  children?: ReactNode
+  message?: ReactNode
+  messageType?: InfoColor
+}
+
+export function DescriptionListItemWithMessage({
+  label,
+  children,
+  message,
+  messageType
+}: DescriptionListWithMessageItemProps) {
+  return (
+    <div
```

Comment:
instead of copying the html from `DescriptionListItem` component you could reuse it here.

```
<DescriptionListItem label={label}>
  {children || <InfoMessage messageType={messageType} message={message}></InfoMessage>}
</DescriptionListItem>
```
Also you'll not need the `'-'` anymore.